### PR TITLE
Date format has been changed

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### Description 
Because of this  PR https://github.com/Katello/katello/commit/c9ad1a640dca56531a592f026e5d1f9d2a686106#diff-d75d6d155e8a47566271ee84888b66caR141 the Date format has changed from 

'shortDate': equivalent to 'M/d/yy' to 'medium': equivalent to 'MMM d, y h:mm:ss a'

@pgagne  as a component owner do you have any issues with this PR?  

### Test Result 

```

Testing started at 4:17 PM ...
/home/okhatavk/Satellite/robottelo/env/bin/python /home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_errata.py::test_end_to_end
Launching py.test with arguments test_errata.py::test_end_to_end in /home/okhatavk/Satellite/robottelo/tests/foreman/ui

2019-07-16 16:17:29 - conftest - DEBUG - Registering custom pytest_configure

2019-07-16 16:17:29 - conftest - DEBUG - Fetching BZs to deselect...

2019-07-16 16:17:49 - conftest - DEBUG - Deselected tests reason: BZ resolution ['1230902', '1479291', '1199150', '1217635', '1214312', '1278917', '1156555', '1347658', '1487317', '1475443', '1147100', '1311113', '1310422', '1204686', '1414821', '1226425']

2019-07-16 16:17:49 - conftest - DEBUG - Deselected tests reason: missing version flag ['1230902', '1489322', '1479291', '1625783', '1199150', '1217635', '1214312', '1278917', '1378442', '1194476', '1156555', '1347658', '1487317', '1581628', '1475443', '1147100', '1311113', '1310422', '1204686', '1436209', '1701118', '1226425', '1682940', '1610309', '1701132']

2019-07-16 16:17:49 - conftest - DEBUG - Deselected tests reason: sat-backlog ['1230902', '1489322', '1479291', '1625783', '1199150', '1217635', '1214312', '1278917', '1378442', '1194476', '1156555', '1347658', '1487317', '1581628', '1475443', '1147100', '1311113', '1310422', '1204686', '1436209', '1701118', '1226425', '1682940', '1610309', '1701132']

============================= test session starts ==============================
platform linux -- Python 3.4.8, pytest-4.0.2, py-1.7.0, pluggy-0.9.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-1.0.2, cov-2.6.12019-07-16 16:17:49 - conftest - DEBUG - Collected 1 test cases

collected 1 item

==================== 1 passed, 8 warnings in 613.49 seconds ====================
```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
@pgagne, @swadeley: Can you please take look on this PR, raised if you have questions.

**Comment by swadeley** [comment]
Hello @omkarkhatavkar , thank you for providing the link to the git commit that caused the change. I have added that info to the bug I raised about date format changes:
  [Bug 1729189 - Date format changed from ISO 8601 to USA format](https://bugzilla.redhat.com/show_bug.cgi?id=1729189)

**Comment by omkarkhatavkar** [comment]
> mmit that ca

@swadeley  Let's wait for that Bug 1729189 to get approved, in that case, we don't want this commit to get merged.

**Comment by swadeley** [comment]
@omkarkhatavkar I think we best merge this now as that bug is not accepted. 
tl;dr Changing the date to absolute time from relative was deliberate. There is no acceptance that time should not be in USA format. Even if the code one day supports date formats based on account or browser settings it would default to USA it seems.
